includes/class_form.php

```diff
@@ -71,11 +71,11 @@ function create_input($name, $properties)
 				break;
 
 			case 'LIST_CHECKBOX':
-			case 'LIST_FLAGS':
```

Comment:
Why you moved this below?

Reply:
Because in the case of a `LIST_FLAGS`, we don't want an array of default values.